integrations/tensorflow/iree_tf_compiler/TFL/LowerGlobalTensors.cpp

```diff
@@ -0,0 +1,162 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Dialect/Util/IR/UtilDialect.h"
+#include "iree/compiler/Dialect/Util/IR/UtilOps.h"
+#include "iree/compiler/Dialect/Util/IR/UtilTypes.h"
+#include "iree/compiler/Utils/ConversionUtils.h"
+#include "iree_tf_compiler/TFL/Passes.h"
+#include "llvm/ADT/StringExtras.h"
+#include "llvm/Support/FormatVariadic.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Support/LLVM.h"
+#include "mlir/Transforms/DialectConversion.h"
+#include "tensorflow/compiler/mlir/lite/ir/tfl_ops.h"
+
+namespace mlir {
+namespace iree_integrations {
+namespace TFL {
+
+class LowerGlobalTensorsPass
+    : public PassWrapper<LowerGlobalTensorsPass, OperationPass<ModuleOp>> {
+ public:
+  void getDependentDialects(DialectRegistry& registry) const override {
+    registry.insert<mlir::TFL::TensorFlowLiteDialect,
+                    iree_compiler::IREE::Util::UtilDialect>();
+  }
+
+  StringRef getArgument() const override {
+    return "iree-tflite-lower-global-tensors";
+  }
+
+  StringRef getDescription() const override {
+    return "Lowers tflite global tensors to IREE flow dialect variables";
+  }
+
+  // Validates that no TFLite frontends ops are in the module.
```

Comment:
Copy/pasted comment? Doesn't seem to describe what this pass is doing.

Reply:
Fixed.